.github/workflows/tests.yml

```diff
@@ -0,0 +1,64 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [7.2, 7.3]
```

Comment:
Why not 7.4? it's officially supported for T3 8-10

Reply:
I just migrated exactly the settings from travis without any (more or less) adjustments.